packages/api/src/config.ts

```diff
@@ -25,3 +25,8 @@ export const PORT = Number.parseInt(env.PORT as string);
 export const REDIS_URL = env.REDIS_URL as string;
 export const SERVER_URL = env.SERVER_URL as string;
 export const USAGE_STATS_ENABLED = env.USAGE_STATS_ENABLED !== 'false';
+export const CACHE_METRICS_TAGS = env.CACHE_METRICS_TAGS !== 'false';
+// deliberately using '||' instead of '??' to avoid empty/falsey values
+// returning as a string since all env values are stringish (to be parsed using ms())
+export const CACHE_METRICS_EXPIRATION =
+  (env.CACHE_METRICS_EXPIRATION as string) || '600s';
```

Comment:
we want this defaults to '600' instead of '600s', right ?

Reply:
Yeah I was trying to decide whether it should be a standard time specifier or number in seconds, let me make it `600` again.